src/ape_compile/_cli.py

```diff
@@ -35,7 +35,8 @@ def cli(cli_ctx, file_paths, use_cache, display_size):
     a project is loaded. You do not have to manually trigger a recompile.
     """
     if not file_paths and cli_ctx.project.sources_missing:
-        cli_ctx.logger.warning("No 'contracts/' directory detected")
+        contracts_dir_name = cli_ctx.project.config.contracts_folder.name
+        cli_ctx.logger.warning(f"No '{contracts_dir_name}/' directory detected")
```

Comment:
Doesn't this happen here now?
https://github.com/ApeWorX/ape/pull/429/files#diff-140ab16c725a530ed14f1900c24c98f2f3fc4b9ec4b96a0c439f0daeaf02de26R74

Reply:
Yeah, this is always `"contracts"` from what I can tell for that reason. It does not hurt to be safe but am fine with removing it too.

Reply:
yeah seems redundant now

Reply:
Okay, I was wrong, so this is needed because you also get this error if the directory exists but is empty.

Could refactor so we check if it empty as config-load time though... Hmmmm...

Reply:
I put it back. I think it is ok for it to be empty at other points ape but just not here.